scripts/kali_fix.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-echo "Running Kali Fix to enable 32bit execution support"
+echo 'Running Kali Fix to enable 32bit execution support'
 export DEBIAN_FRONTEND=noninteractive
 dpkg --add-architecture i386
 apt-get update -q
-apt-get install -q -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" libstdc++6:i386 zlib1g:i386 libncurses5:i386
+apt-get install -q -y -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' libstdc++6:i386 zlib1g:i386 libncurses5:i386
```

Comment:
@superpoussin22 Is this tested.Bash has different meaning for `'` vs `"`

Reply:
@ajin yes :)
`Step 19/34 : RUN ./scripts/kali_fix.sh
 ---> Running in 9037ac46925d
Running Kali Fix to enable 32bit execution support
Get:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Hit:2 http://archive.ubuntu.com/ubuntu bionic InRelease
Hit:3 http://archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu bionic-backports InRelease
Get:5 http://archive.ubuntu.com/ubuntu bionic/multiverse i386 Packages [177 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic/main i386 Packages [1,328 kB]
Get:7 http://archive.ubuntu.com/ubuntu bionic/restricted i386 Packages [13.5 kB]
Get:8 http://archive.ubuntu.com/ubuntu bionic/universe i386 Packages [11.3 MB]
Get:9 http://security.ubuntu.com/ubuntu bionic-security/multiverse i386 Packages [4,349 B]
Get:10 http://security.ubuntu.com/ubuntu bionic-security/universe i386 Packages [716 kB]
Get:11 http://security.ubuntu.com/ubuntu bionic-security/main i386 Packages [435 kB]
Get:12 http://security.ubuntu.com/ubuntu bionic-security/restricted i386 Packages [5,385 B]
Get:13 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse i386 Packages [7,407 B]
Get:14 http://archive.ubuntu.com/ubuntu bionic-updates/universe i386 Packages [1,223 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic-updates/restricted i386 Packages [10.7 kB]
Get:16 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 Packages [728 kB]
Get:17 http://archive.ubuntu.com/ubuntu bionic-backports/universe i386 Packages [3,926 B]
Get:18 http://archive.ubuntu.com/ubuntu bionic-backports/main i386 Packages [2,493 B]
Fetched 16.0 MB in 23s (697 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  gcc-8-base:i386 libc6:i386 libgcc1:i386 libgpm2:i386 libtinfo5:i386
Suggested packages:
  glibc-doc:i386 locales:i386 gpm:i386
The following NEW packages will be installed:
  gcc-8-base:i386 libc6:i386 libgcc1:i386 libgpm2:i386 libncurses5:i386
  libstdc++6:i386 libtinfo5:i386 zlib1g:i386
0 upgraded, 8 newly installed, 0 to remove and 4 not upgraded.
Need to get 3,304 kB of archives.
After this operation, 14.1 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 gcc-8-base i386 8.3.0-6ubuntu1~18.04.1 [18.7 kB]
Get:2 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 libgcc1 i386 1:8.3.0-6ubuntu1~18.04.1 [48.1 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic/main i386 libc6 i386 2.27-3ubuntu1 [2,551 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 libtinfo5 i386 6.1-1ubuntu1.18.04 [80.4 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 libncurses5 i386 6.1-1ubuntu1.18.04 [101 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic/main i386 zlib1g i386 1:1.2.11.dfsg-0ubuntu2 [58.2 kB]
Get:7 http://archive.ubuntu.com/ubuntu bionic-updates/main i386 libstdc++6 i386 8.3.0-6ubuntu1~18.04.1 [431 kB]
Get:8 http://archive.ubuntu.com/ubuntu bionic/main i386 libgpm2 i386 1.20.7-5 [15.8 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 3,304 kB in 20s (164 kB/s)
Selecting previously unselected package gcc-8-base:i386.
(Reading database ... 41256 files and directories currently installed.)
Preparing to unpack .../0-gcc-8-base_8.3.0-6ubuntu1~18.04.1_i386.deb ...
Unpacking gcc-8-base:i386 (8.3.0-6ubuntu1~18.04.1) ...
Selecting previously unselected package libgcc1:i386.
Preparing to unpack .../1-libgcc1_1%3a8.3.0-6ubuntu1~18.04.1_i386.deb ...
Unpacking libgcc1:i386 (1:8.3.0-6ubuntu1~18.04.1) ...
Selecting previously unselected package libc6:i386.
Preparing to unpack .../2-libc6_2.27-3ubuntu1_i386.deb ...
Unpacking libc6:i386 (2.27-3ubuntu1) ...
Selecting previously unselected package libtinfo5:i386.
Preparing to unpack .../3-libtinfo5_6.1-1ubuntu1.18.04_i386.deb ...
Unpacking libtinfo5:i386 (6.1-1ubuntu1.18.04) ...
Selecting previously unselected package libncurses5:i386.
Preparing to unpack .../4-libncurses5_6.1-1ubuntu1.18.04_i386.deb ...
Unpacking libncurses5:i386 (6.1-1ubuntu1.18.04) ...
Selecting previously unselected package zlib1g:i386.
Preparing to unpack .../5-zlib1g_1%3a1.2.11.dfsg-0ubuntu2_i386.deb ...
Unpacking zlib1g:i386 (1:1.2.11.dfsg-0ubuntu2) ...
Selecting previously unselected package libstdc++6:i386.
Preparing to unpack .../6-libstdc++6_8.3.0-6ubuntu1~18.04.1_i386.deb ...
Unpacking libstdc++6:i386 (8.3.0-6ubuntu1~18.04.1) ...
Selecting previously unselected package libgpm2:i386.
Preparing to unpack .../7-libgpm2_1.20.7-5_i386.deb ...
Unpacking libgpm2:i386 (1.20.7-5) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
Setting up gcc-8-base:i386 (8.3.0-6ubuntu1~18.04.1) ...
Setting up libc6:i386 (2.27-3ubuntu1) ...
Setting up libgcc1:i386 (1:8.3.0-6ubuntu1~18.04.1) ...
Setting up libtinfo5:i386 (6.1-1ubuntu1.18.04) ...
Setting up libgpm2:i386 (1.20.7-5) ...
Setting up zlib1g:i386 (1:1.2.11.dfsg-0ubuntu2) ...
Setting up libstdc++6:i386 (8.3.0-6ubuntu1~18.04.1) ...
Setting up libncurses5:i386 (6.1-1ubuntu1.18.04) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
`